be/src/util/json.h

```diff
@@ -29,6 +29,9 @@ enum JsonType {
     JSON_OBJECT = 5,
 };
 
+// Maximum length of JSON string is 16MB
+constexpr size_t kJSONLengthLimit = 16 << 20;
```

Comment:
Do we need to make it to be configurable？

Reply:
It's a defense for system safety, cause we could not support a huge JSON string

If someday we could support a huge JSON string, just remove this limitation.

Anyway, nobody want to configure this limitation.